3.11/aql/fundamentals-syntax.md

```diff
@@ -255,8 +255,23 @@ FOR doc IN `my-coll`
 
 The collection `my-coll` has a dash in its name, but `-` is an arithmetic
 operator for subtraction in AQL. The backticks escape the collection name to
-refer to the collection correctly. Note that quoting the name with `"` or `'`
-is not possible for collections.
+refer to the collection correctly.
+
+If you use extended collection and View names
+([`--database.extended-names` startup option](../programs-arangod-options.html#--databaseextended-names)),
+they may contain spaces, or non-ASCII characters such as Japanese or Arabic
+letters, emojis, letters with accentuation, and other Unicode characters.
+Escaping is required in these cases, too:
```

Comment:
Nit: I don't think that _escaping_ is required here, but rather _quoting_.

Reply:
Hm, I think it was already called escaping further up in the existing text. Need to check where else we might say escaping but mean quoting...

Reply:
Definitely possible. But I think "quoting" would be the more correct term here.